abdulfataiaka/hackworld
# Setup development environment

## Overview

- Implement `gulp task runner`, `webpack bundler` and `babel transpiler`
- Create convenient scripts for development

## How to test

- Pull branch
- Notice the scripts `watch` and `setup` in `package.json`
- Run `npm run setup` to install dependencies and start the server
- Visit `http://127.0.0.1:8080` to see app run given the screenshot below
 
<img width="1440" alt="Screen Shot 2020-01-14 at 8 04 26 PM" src="https://user-images.githubusercontent.com/32123313/72373974-ade63500-3709-11ea-9dbf-54aa3f8e1089.png">

## General Note

- You can run `npm run watch` to watch file changes for rebuild
